eslint.config.js

```diff
@@ -25,4 +25,9 @@ export default tseslint.config(
 			'**/__snapshots__/**',
 		],
 	},
+	{
+		rules:{
+			'no-console':'error'
+		}
+	},
```

Comment:
Almost there! 🚀 

just one last change - can you please put it together with the `ignores` key?